build.psm1

```diff
@@ -597,13 +597,19 @@ function Publish-PSTestTools {
 
     Find-Dotnet
 
+    $tools = "$PSScriptRoot/test/tools/EchoArgs","$PSScriptRoot/test/tools/CreateChildProcess"
     # Publish EchoArgs so it can be run by tests
-    Push-Location "$PSScriptRoot/test/tools/EchoArgs"
-    try {
-        dotnet publish --output bin
-    } finally {
-        Pop-Location
+
+    foreach ($tool in $tools)
+    {
+        Push-Location $tool
+        try {
+            dotnet publish --output bin
```

Comment:
This does not seem like a good solution to your test issue. Is there a better way to handle it? Packaging scoops up all files in the publish directory, so these tools will get shipped along with PowerShell Core during release.